src/wdio/browser.ts

```diff
@@ -12,9 +12,21 @@ export class Browser {
 
 
     // Keyboard special keys
+    public static async pressEsc(): Promise<void> {
+        return browser.keys(['Escape']);
+    }
```

Comment:
A picky one I know, but just to keep the code styling on this file, I miss a blank line between the added methods.